roles/swift-proxy/templates/etc/swift/proxy-server.conf

```diff
@@ -50,8 +50,8 @@ use = egg:swift#name_check
 use = egg:swift#list_endpoints
 
 [filter:keystoneauth]
-operator_roles = Member,_member_,admin
-reseller_prefix =
+operator_roles = Member,_member_,admin,cloud_admin,project_admin
```

Comment:
@craigtracey: Are you recommending only the new changes (cloud/project admin) to be configurable or entire operator_roles/reseller_admin_role?